services/management/annotation.go

```diff
@@ -0,0 +1,85 @@
+// pmm-managed
+// Copyright (C) 2017 Percona LLC
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+package management
+
+import (
+	"context"
+	"strings"
+	"time"
+
+	"github.com/percona/pmm/api/managementpb"
+	"gopkg.in/reform.v1"
+
+	"github.com/percona/pmm-managed/models"
+)
+
+type grafanaClient interface {
```

Comment:
@JiriCtvrtka, as Nurlan mentioned it is more convenient to move this interface to https://github.com/percona/pmm-managed/blob/PMM-2.0/services/management/deps.go

and add a similar line as https://github.com/percona/pmm-managed/blob/8a6d53d26120ccdb1314bf234405092530afa4e1/services/management/deps.go#L29

then `make gen` will generate mock as https://github.com/percona/pmm-managed/blob/PMM-2.0/services/management/mock_checks_service_test.go
to include in `annotation_test.go`